templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -0,0 +1,151 @@
+const navigation = document.querySelector('.p-navigation--sliding');
```

Comment:
Could all these variables be moved inside the `initNavigationSliding` function? So they don't pollute global scope and are initialised when the function is called.